src/protocols/secure_channel/SessionResumptionStorage.h

```diff
@@ -39,13 +30,10 @@ namespace chip {
  *     * <FabricIndex, PeerNodeId>   => <ResumptionId, ShareSecret, PeerCATs>
  *     * <ResumptionId>              => <FabricIndex, PeerNodeId>
  */
-class SessionResumptionStorage
+class SessionResumptionStorage : public AbstractSessionResumptionStorage
```

Comment:
This whole PR would have been smaller/simpler with just `SessionResumptionStorage` renamed to `DefaultResumptionStorage` (including filename) and the interface just named `SessionResumptionStorage` instead of `AbstractSessionResumptionStorage`.

Reply:
Yes, it would have been.  What I am doing is correcting that.  The intent here is to define the minimal public interface, and this does not and should not be dependent on internal table layout.

Reply:
Ah, sorry I misunderstood.  Yes, that is a much lower line count change.

Reply:
Done.